### DISCUSSION TITLE
Pythone pwn tools error

### DISCUSSION THREAD

**Discussion by destructB** [community/community#134217] [Programming Help]
### Select Topic Area

Question

### Body

┌──(root㉿kali)-[/vsftpd_2.3.4_Exploit]
└─# python3 exploit.py (IP adress) 21 id

**[ERROR]** Usage: exploit.py IP PORT(optional)
Traceback (most recent call last):
  File "/vsftpd_2.3.4_Exploit/exploit.py", **line 37, in <module>
    error**(f"Usage: {sys.argv[0]} IP PORT(optional)")
  File "/usr/local/lib/python3.11/dist-packages/pwnlib/log.py", line **439, in error**
    raise PwnlibException(message % args)
pwnlib.exception.PwnlibException: Usage: exploit.py IP PORT(optional)

I need help to resolve this problem. The version of Pwntools v4.12.0.
<img width="745" alt="pyerror" src="https://github.com/user-attachments/assets/d1f04e2d-a85b-4a76-a1eb-21101ce71d78">

**Comment by LiteBrite82** [comment]
Thanks for posting in the GitHub Community, @destructB ! 

We’ve moved your post to our Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
